README.md

```diff
@@ -162,6 +163,21 @@ config {
 }
 ```
 
+### auth
+
+To authenticate pulling private image, user can setting the `auth` stanza with `username` and `password` fields.
+
+```
+config {
+  auth {
+    username = "user"
+    password = "pass"
+  }
+}
+```
+
+if the credential is an identity token without `username`, set the identity token to the `password` field.
```

Comment:
nitpick: s/if/If

Reply:
I generated a docker access token for one of my private repositories (image) using https://docs.docker.com/docker-hub/access-tokens/
However, when I tried to run the nomad job with `auth` stanza, without specifying the `username` and just the `token`, it failed for me!

Does this work? Have you tried this with an example?

Reply:
we can see that dockhub needs `--username` with access-token in the link. so it should not work without a username.

I do not test this scene for i don't know how to get a idnetity token.  this scene is reference from https://github.com/containerd/containerd/blob/ab963e1cc16a845567a0e3e971775c29c701fcf8/pkg/cri/server/image_pull.go#L194

May be we should remove it?

Reply:
Yeah, let's remove this line for now, otherwise it might confuse the users.
We can add support for this (Authenticating just using the `identity token` without `username`) once we have a better way to test this scenario.